ISISNeutronMuon/MDANSE
# Improved labels, corrected text output

**Description of work**
This PR mainly deals with labels, docstrings and naming in MDANSE. Additionally, problems with analysis output in text format have been corrected so that text output works again.

**Fixes**
1. Text output works again,
2. Files in text output have correct creation time set,
3. ASCIIFormat is now TextFormat,
4. Renamed 'monoprocessor' and 'multiprocessor' to (more accurate) 'single-core' and 'multicore'.
5. Added a docstring to RigidBodyTrajectory,
6. Activated tooltips in the GUI for several widgets which were missing them before.

**To test**
1. Load a trajectory,
2. Check if all the action dialogs are created correctly. Look at the widget tooltips in some of the dialogs,
3. Run an analysis using TextFormat as output,
4. Check if the files in the resulting .tar archive are created correctly and with today's timestamp.
